pkgs/tools/networking/wg-netmanager/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, darwin, wireguard-go }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wg-netmanager";
+  version = "0.3.8";
+
+  src = fetchFromGitHub {
+    owner = "gin66";
+    repo = "wg_netmanager";
+    rev = "wg_netmanager-v${version}";
+    sha256 = "1fbrgn1hnygfy337r52dhrq8h2lzdr0b675ys738ibiwwm6s558z";
+  };
+
+  # related to the Cargo.lock file
+  cargoSha256 = "0wj6rlj5iyzgx5xxzd4mmza2kiwfnpyi918w7gb03hqppzd2xkrr";
+
+  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
+
+  # Test 01 tries to create a wireguard interface, which requires sudo.
+  doCheck = false;
```

Comment:
This completely disables tests. If the test run with `checkFlags = "--test 02_";` you should set this to `doCheck = true`;

Reply:
have set doCheck to true